src/ot2rec_report/templates/report_aretomo_align.ipynb

```diff
@@ -50,10 +50,13 @@
     "    with open(mc_yaml_file, \"r\") as f:\n",
     "        params_mc = yaml.load(f.read(), Loader=yaml.FullLoader)\n",
     "\n",
-    "    aretomo_align_yaml_file = f\"{proj_name}_aretomo_align.yaml\"\n",
-    "    if os.path.exists(aretomo_align_yaml_file) is False:\n",
-    "        raise FileNotFoundError(\"AreTomo align yaml file not found, pixel size cannot be extracted\")\n",
-    "    with open(aretomo_align_yaml_file, \"r\") as f:\n",
+    "    aretomo_yaml_file = f\"{proj_name}_aretomo_align.yaml\"\n",
+    "    if os.path.exists(aretomo_yaml_file) is False:\n",
+    "        if os.path.exists(f\"{proj_name}_aretomo_align-recon.yaml\") is True:\n",
+    "            aretomo_yaml_file = f\"{proj_name}_aretomo_align-recon.yaml\"\n",
+    "        else:\n",
+    "            raise FileNotFoundError(\"AreTomo align or align-recon yaml file not found, pixel size cannot be extracted\")\n",
```

Comment:
- I'd move the double-file check to the front - if neither file exists there would be no points doing further checks
- The nested if statements can be contracted to `if (not os.path.exists(aretomo_yaml_file)) and (os.path.exists(f\"{proj_name}_aretomo_align-recon.yaml\"))`. 
- I think a yet better way would be 
```
if os.path.exists(aretomo_yaml_file):
   do something
else:
   do something else
```
Because logically you know they can't both be absent, then the two possibilities: one of them exists or both exist, where in the second case, the `else` statement wouldn't be run because the `if` statement is already true.

Reply:
The second if statement swaps the `aretomo_yaml_file` variable to be the align-recon yaml file created by AreTomo mode 2 if mode 2 is run instead of mode 0. 

Technically in this version both could be absent if the user doesn't actually run the process, since the sections in the report are determined by those which are selected from the magicgui in the beginning. In the future we should definitely put in more checks to make sure that the section is only added if the correct files are present and the section is selected in magicgui.

For now, if the aretomo yaml files that are required don't exist then there will just be an error and the rest of the notebook proceeds with the next sections